native/markdown/rules.react.js

```diff
@@ -33,12 +44,9 @@ export default function rules(styles: StyleSheetOf<MarkdownStyles>) {
         const onPressLink = () => {
           Linking.openURL(node.target);
         };
-        state.linkPresent = true;
-        const innerNode = output(node.content, state);
-        state.linkPresent = false;
```

Comment:
Since we're setting the `textStyle` in the parent again, we don't need `linkPresent`